controllers/openstackcluster_controller.go

```diff
@@ -259,7 +259,7 @@ func deleteBastion(scope scope.Scope, cluster *clusterv1.Cluster, openStackClust
 }
 
 func reconcileNormal(scope scope.Scope, cluster *clusterv1.Cluster, openStackCluster *infrav1.OpenStackCluster) (ctrl.Result, error) {
-	scope.Logger().Info("Reconciling Cluster")
+	scope.Logger().Info("Reconciled Cluster")
```

Comment:
I think this is not correct, as the reconcile has not happened yet. You might log `Reconciled cluster` when the reconcile completes.